pages/data-providers/[data-provider-id]/deposit-compliance.jsx

```diff
@@ -81,6 +81,31 @@ class PublicationDateColumn extends Table.Column {
   }
 }
 
+const getComplianceLevelNumberProps = (complianceLevel) => {
+  if (
+    complianceLevel === null ||
+    (complianceLevel > 0 && complianceLevel < 100)
+  ) {
+    return {
+      value: 'Loading...',
+      append: '%',
+      caption: 'non-compliant',
+    }
+  }
+  if (complianceLevel === 0) {
+    return {
+      value: 'All',
+      append: '',
+      caption: 'outputs are non-compliant',
+    }
+  }
+  return {
+    value: 'All',
+    append: '',
+    caption: 'outputs are compliant',
+  }
```

Comment:
We cannot say this. Our compliance level is an approximation. We could only say _We could not detect non-compliant outputs_ or something like that